Fork and solve error oracle when you execute getColumnListing

mfrancois opened the pull request:
When you want to get the columns name of table with laravel you have to do it like this :

```
 \Schema::getColumnListing($this->model->getTable());

 ```

When we do that you have this error

```
Error Code : 1008
Error Message : ORA-01008: not all variables bound
Position : 0
Statement : select column_name from user_tab_columns where table_name = upper(:p0) and column_name = upper(:p1)
Bindings : []
(SQL: select column_name from user_tab_columns where table_name = upper(?) and column_name = upper(?))
```

This fix solve this error

rafael-renan-pacheco commented:
Hi @mfrancois!

The problem in indeed in `compileColumnExists`, but your query does not work, since it is comparing `column_name` with the `$table` param.

If you look at MySqlGrammar, PostgressGrammar, SQLiteGrammar, and SqlServerGrammar, `compileColumnExists` does not compare any columns, it just compare the table name.

So, the solution should be something like:

```php
public function compileColumnExists($table)
{
    return "select column_name from user_tab_columns where table_name = upper('$table')";
}
```

After that, you need to fix the test assert, and remove that new line in ".gitignore", since ".idea" does not need to be added here, it belongs just to phpstorm projects.

mfrancois commented:
Hi @rafael-renan-pacheco I changed the query and the unit testing to make sure that work properly.
I removed the .idea on .gitignore.

Regard,
Max

yajra commented:
Thanks guys, released on v4.2.2. I also implemented the fix on lower versions. :beers:
